verilog/analysis/descriptions.h

```diff
@@ -20,20 +20,28 @@
 #define VERIBLE_VERILOG_ANALYSIS_DESCRIPTIONS_H_
 
 #include <string>
+#include <vector>
 
 #include "absl/strings/string_view.h"
 
 namespace verilog {
 namespace analysis {
 
-enum class DescriptionType {
-  kHelpRulesFlag,  // emit CLI help text
-  kMarkdown,       // emit Markdown
+using LintRuleId = absl::string_view;
+
+struct LintConfigParameterDescriptor {
+  absl::string_view name;
+  std::string default_value;
+  std::string description;
 };
 
-// Returns a string with the appropriate wrappers to denote it is code in the
-// declared DescriptionType documentation.
-std::string Codify(absl::string_view, DescriptionType);
+struct LintRuleDescriptor {
+  LintRuleId name;             // ID/name of the rule.
+  absl::string_view topic;     // section in style-guide
+  absl::string_view dv_topic;  // section in style-guide
```

Comment:
wonder if we should comment that these really expect constexpr string literals

Reply:
mmh, I think the chance is low that someone is assembling strings there (these are short constants always), and if they do, they haven't looked at the struct (and then, comment) anyway, seeing the `std::string_view` ...

So - probably not.